src/roles/common/tasks/check-predeploy-prereq.yml

```diff
@@ -49,6 +49,12 @@
 
   when: not vsd_inplace_upgrade | default(false)
 
+- name: Check the image path for portal container
```

Comment:
We should do the same md5 file deletion as from #1658 here.

Reply:
Yeah Once Urvish is done I will add those changes

Reply:
Done updated